pkgs/development/guile-modules/guile-ssh/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, guile, libssh
+, autoreconfHook, pkg-config, texinfo, which
+}:
+
+stdenv.mkDerivation rec {
+  pname = "guile-ssh";
+  version = "0.12.0";
+
+  src = fetchurl {
```

Comment:
Any reason you didn't use `fetchFromGitHub` here?

Reply:
Same as ever